test/helpers/types/types.go

```diff
@@ -44,12 +45,14 @@ type Docker struct {
 type Kubernetes struct {
 	Annotations       map[string]string `json:"annotations,omitempty"`
 	ContainerName     string            `json:"container_name,omitempty"`
+	ContainerID       string            `json:"container_id,omitempty"`
 	NamespaceName     string            `json:"namespace_name,omitempty"`
 	PodName           string            `json:"pod_name,omitempty"`
-	ContainerImage    string            `json:"container_image,omitempty"`
-	ContainerImageID  string            `json:"container_image_id,omitempty"`
+	PodNodeName       string            `json:"pod_node_name,omitempty"`
```

Comment:
Why do we need pod node name?   how is this different from "host"?

Reply:
`pod_node_name` contains value from `spec.nodeName`